Update charter to reflect shipping RFC5033bis (#2)
ietf-wg-ccwg/wg-materials: charter.md

> Congestion control is the process a network sender uses to determine the rate at which to send data. A congestion control algorithm balances sending sufficient data to make meaningful progress for a user with avoiding filling buffers in the network and overloading the network path. Better congestion control algorithms can help improve application performance, especially latency under load, while safeguarding overall network performance for all senders.

Reviewer: It would be nice to also briefly say why congestion control standardization is valuable, since it is not generally an interoperability concern.

Reviewer 2: I love that latency under load is mentioned, but don't think "especially" is a good choice of words. "such as", or "for example" might be better?

Reviewer 3: ```suggestion
Congestion control is the process a network sender uses to determine the rate at which to send data. A congestion control algorithm balances sending sufficient data to make meaningful progress for a user with avoiding filling buffers in the network and overloading the network path. Better congestion control algorithms can help improve application performance, such as latency under load, while safeguarding overall network performance for all senders.
```
Getting back to this PR, "such as" sounds good to me

Reviewer 3: > It would be nice to also briefly say why congestion control standardization is valuable, since it is not generally an interoperability concern.

Agreed. We could reuse text from the Introduction of 5033bis here:

> […] a specification for a congestion control algorithm provides a number of advantages:
>
> It can help implementers, operators, and other interested parties develop a shared understanding of how the algorithm works and how it is expected to behave in various scenarios and configurations.
>
> It can help potential contributors understand the algorithm, which can make it easier for them to suggest improvements and/or identify limitations. Furthermore, the specification can help multiple contributors align on a consensus change to the algorithm.
> 
> A specification that is accessible to anyone can circumvent the issue that some implementers may be unable to read open source reference implementations due to the constraints of some open source licenses.

What does everyone else think?

Author: Updated in [46b0462](https://github.com/ietf-wg-ccwg/wg-materials/pull/2/commits/46b04624062cfcef62859e3f65895d5d7ebc57b8)!

Congestion control is the process a network sender uses to determine the rate at which to send data. A congestion control algorithm balances sending sufficient data to make meaningful progress for a user with avoiding filling buffers in the network and overloading the network path. Better congestion control algorithms can help improve application performance, such as latency under load, while safeguarding overall network performance for all senders.

> Standardizing a congestion control algorithm can help implementers, operators, and other interested parties develop a shared understanding of how the algorithm works, how it is expected to behave in various scenarios and configurations, and can make it easier for them to suggest improvements and identify limitations.

Reviewer 3: ```suggestion
Specifying a congestion control algorithm can help implementers, operators, and other interested parties develop a shared understanding of how the algorithm works, how it is expected to behave in various scenarios and configurations, and can make it easier for them to suggest improvements and identify limitations.
```
Nit, but not all specifications are standards, and I catch myself most times I conflate them... :D

Author: Oh great catch, thank you!

Congestion control algorithm proponents now often have the opportunity to test and deploy at scale without IETF review. The set of protocols using these algorithms has spread beyond TCP and SCTP to include DCCP, QUIC, and beyond. There is more interest in specialized use cases such as data centers and real-time protocols. The community has gained much more experience with indications of congestion beyond packet loss.

> The Congestion Control Working Group analzyes impediments to congestion control work occurring in the IETF and generalizes transports from TCP to all of the relevant transport protocols. It has revised [RFC 5033](https://datatracker.ietf.org/doc/rfc5033/) (5033bis) to encourage IETF review of congestion control proposals and standardization of mature congestion control algorithms.

Reviewer 3: Typo:
```suggestion
The Congestion Control Working Group analyzes impediments to congestion control work occurring in the IETF and generalizes transports from TCP to all of the relevant transport protocols. It has revised [RFC 5033](https://datatracker.ietf.org/doc/rfc5033/) (5033bis) to encourage IETF review of congestion control proposals and standardization of mature congestion control algorithms.
```
